src/plugins/visualizations/server/embeddable/visualize_embeddable_factory.ts

```diff
@@ -36,6 +37,15 @@ const byValueRemoveDefaultIndexPatternAndTimeFieldFromTSVBModel = (state: Serial
   };
 };
 
+const byValueAddEmptyValueRuleForSavedObjectsWithLessAndGreaterThenZeroRule = (
```

Comment:
This is a nit but I would make the function names smaller:
byValueAddEmptyValueColorRule, commonAddEmptyValueColorRule etc